internal/backends/qt/qt_widgets/spinbox.rs

```diff
@@ -289,6 +291,12 @@ impl Item for NativeSpinBox {
         let data = this.data();
         let active_controls = data.active_controls;
         let pressed = data.pressed;
+        let horizontal_alignment = match this.horizontal_alignment() {
+            // TextHorizontalAlignment to Qt::AligmentFlag
+            TextHorizontalAlignment::Left => 1,
```

Comment:
Would it be possible to avoid the use of hard-coded constants here and map this like in other places like this? https://github.com/slint-ui/slint/blob/d8a62ab41e55d724e483e0ddd5371c2f7dfe1f1f/internal/backends/qt/qt_window.rs#L686